src/main/webapp/assets/feedStyle.css

```diff
@@ -115,3 +106,27 @@
   color: black;
   margin-top: 1.25rem;
 }
+
+@media screen and (max-width: 678px) {
+  #col-container {
+    margin-left: 5%;
+    margin-right: 5%;
+    width: 90%;
+  }
+
+  #map {
+    height: 40vh;
+  }
+
+  #all-posts {
+    flex: 1 0 0;
+    height: 65vh;
+    margin-bottom: 5px;
+    margin-left: auto;
+    margin-right: 0;
+    overflow-y: auto;
+    padding-top: 30px !important;
+    padding-left: 0 !important;
+    padding-right: 0 !important;
```

Comment:
I would recommend not using `!important` without careful thought (see https://uxengineer.com/css-specificity-avoid-important-css/). If you're sure that this is the best way to implement this, though, maybe add a comment explaining why?

Reply:
Gotcha, good catch! Just updated the CSS!